lxd/networks.go

```diff
@@ -40,17 +40,17 @@ var networksCmd = APIEndpoint{
 	Path: "networks",
 
 	Get:  APIEndpointAction{Handler: networksGet, AccessHandler: allowAuthenticated},
-	Post: APIEndpointAction{Handler: networksPost},
+	Post: APIEndpointAction{Handler: networksPost, AccessHandler: allowProjectPermission("networks", "manage-networks")},
 }
 
 var networkCmd = APIEndpoint{
 	Path: "networks/{name}",
 
 	Delete: APIEndpointAction{Handler: networkDelete},
 	Get:    APIEndpointAction{Handler: networkGet, AccessHandler: allowAuthenticated},
```

Comment:
@stgraber The storage volume routes check for a permission of `view` here, should I also add that here, and the GET request above?

Reply:
Yeah, that will properly restrict it to projects and make it such that someone who's allowed just read-only access can still access it.

Reply:
OK done.